static/js/impala/stats/manager.js

```diff
@@ -518,9 +520,14 @@ z.StatsManager = (function() {
             key = parts[0];
         parts = parts.slice(1);
 
+        // Fenix app versions are build IDs so we need a prefix for end users.
+        var versionPrefix = metric === 'apps' &&
+            key === '{aa3c5121-dab2-40e2-81ca-7ea25febc110}' &&
```

Comment:
code-coding-tastic .... if this is just a temporary fix then fine.  If it's longer term I'd be concerned about this - would we even look in js code for something fenix application specific?